lib/Doctrine/Search/File.php

```diff
@@ -53,12 +53,14 @@ public function __construct(array $options = array())
             $this->_options['fields'] = array('url', 'content');
         }
 
-        $this->initialize($table);
+        if (isset($table)) {
+            $this->initialize($table);
```

Comment:
Only do this if `$table` var is set (the initialize method this calls has a typehint on the first param that would cause this to throw a fatal error if ever called with `null` for `$table` I believe).